src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorage/AzureTableStorageSink.cs

```diff
@@ -39,20 +38,6 @@ public class AzureTableStorageSink : ILogEventSink
         /// <param name="storageAccount">The Cloud Storage Account to use to insert the log entries to.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
         /// <param name="storageTableName">Table name that log entries will be written to. Note: Optional, setting this may impact performance</param>
-        public AzureTableStorageSink(
```

Comment:
It ended up being unnecessary since the `keyGenerator` parameter is nullable.....and when it's null, you'll get a `new DefaultKeyGenerator()`